src/Compiler/Microsoft.AspNetCore.Razor.Language/src/CodeGeneration/CodeWriter.cs

```diff
@@ -148,7 +146,7 @@ public CodeWriter Write(string value, int startIndex, int count)
     }
 
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    internal CodeWriter WriteCore(string value, int startIndex, int count)
+    private CodeWriter WriteCore(string value, int startIndex, int count)
```

Comment:
Did you consider just having this take `ReadOnlyMemory<char>` to avoid the entire `start`, `count` parts?

Reply:
That will come as part of a later change. I wanted to keep the bug fix separate.